app/src/main/java/ru/vssemikoz/newsfeed/models/News.java

```diff
@@ -0,0 +1,50 @@
+package ru.vssemikoz.newsfeed.models;
+
+import java.net.URL;
+import java.util.Date;
+
+public class News {
+    private String author, title, description, content;
+    private URL url, urlToImage;
+    private Date publishedAt;
+
+    News( String author, String title, String description, String content, URL url, URL urlToImage, Date publishedAt){
```

Comment:
I think we can use default constructor with a set of getters/setters as JSON/Database libs will probably generate required constructors themselves.